source/renderer/app/components/wallet/transactions/Transaction.js

```diff
@@ -194,33 +194,26 @@ export default class Transaction extends Component<Props> {
       ? transactionStates.PENDING
       : state;
 
-    const componentStyles = classNames([
-      styles.component,
-      isFailedTransaction ? styles.failed : null,
-      isExpanded ? 'Transaction_expanded' : null,
-    ]);
+    const componentStyles = classNames(styles.component, {
+      [styles.failed]: isFailedTransaction,
+      Transaction_expanded: isExpanded,
+    });
 
-    const contentStyles = classNames([
-      styles.content,
-      isLastInList ? styles.last : null,
-      isExpanded ? styles.contentExpanded : null,
-    ]);
+    const contentStyles = classNames(styles.content, {
+      [styles.last]: isLastInList,
+      [styles.contentExpanded]: isExpanded,
+    });
 
-    const detailsStyles = classNames([
-      styles.details,
-      canOpenExplorer ? styles.clickable : null,
-      isExpanded ? styles.detailsExpanded : styles.detailsClosed,
-    ]);
+    const detailsStyles = classNames(styles.details, {
+      [styles.clickable]: canOpenExplorer,
+      [styles.detailsExpanded]: isExpanded,
```

Comment:
@lrosa007 this line is wrong... It should read:
```
[styles.detailsExpanded]: isExpanded,
[styles.detailsClosed]: !isExpanded,
```

Reply:
Thanks, fixing ASAP

Reply:
@nikolaglumac ok it's addressed